test/knex.test.js

```diff
@@ -259,6 +259,30 @@ describe('transaction', function () {
 
     throw new Error('expected the call to throw')
   })
+
+
+  it('adopt', async () => {
+    const s0 = await si.entity.adopt()
```

Comment:
Hmm, I don't think so, because we call the transaction through the flag:
`const si = makeSenecaForTest(
    DbConfigPG, 
    {entity_opts: { transaction: {active:true} }
  })`